src/routes/explore/explore.component.tsx

```diff
@@ -36,22 +45,58 @@ export const Explore = () => {
   const radarRings = getRadarRings(rings);
   const radarTeams = getRadarTeams(teams);
 
-  const updateActiveQuadrant = (newQuadrant: number) => {
+  const currentTechnologies = zoomedQuadrant ? zoomedTechnologies : radarTechnologies;
+
+  useEffect(() => {
+    if (!isEmpty(radarQuadrants)) {
+      const quadrantForArea = radarQuadrants.find((quadrant) => quadrant.name === areaValue);
+      setActiveQuadrant(quadrantForArea ? quadrantForArea.position : null);
+    }
+  }, [areaValue, radarQuadrants]);
+
+  const filterTechnologies = () => {
```

Comment:
I'm thinking about moving this function into utils.

Reply:
yep, we should do it. 

In general the best attitude is to keep the file as simple and clean as possible.